markdown/blockprocessors.py

```diff
@@ -314,6 +308,22 @@ class OListProcessor(BlockProcessor):
     # List of allowed sibling tags.
     SIBLING_TAGS = ['ol', 'ul']
 
+    def __init__(self, parser):
+        BlockProcessor.__init__(self, parser)
```

Comment:
Are you targeting compatibility with Python 2? If so, I'm not sure `super()` would work in this context, because using it [requires new style classes](https://docs.python.org/2/library/functions.html#super).


Also, a call to `super()` seems like it would be inconsistent in the code base. In `ListIndentProcessor.__init__(...)` method (line 144 in `markdown/blockprocessors.py`), there is a call to `BlockProcessor.__init__(...)`.